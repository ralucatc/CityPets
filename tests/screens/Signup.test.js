import React from 'react';
import renderer from 'react-test-renderer';
import Signup from '../../screens/signup';

describe('<Signup />', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Signup />).toJSON();
        expect(tree).toMatchSnapshot();
      });
  });