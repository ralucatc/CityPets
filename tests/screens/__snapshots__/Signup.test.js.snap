// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Signup /> renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "display": "flex",
      "flex": 1,
      "flexDirection": "column",
      "justifyContent": "center",
      "padding": 35,
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "flex": 1,
        "justifyContent": "flex-start",
        "marginBottom": 100,
        "marginTop": -150,
        "maxHeight": 250,
        "maxWidth": 350,
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": undefined,
            "width": undefined,
          },
          undefined,
        ]
      }
    />
  </View>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Name"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "center",
        "borderBottomWidth": 1,
        "borderColor": "#ccc",
        "marginBottom": 15,
        "paddingBottom": 15,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="Email"
    rejectResponderTermination={true}
    style={
      Object {
        "alignSelf": "center",
        "borderBottomWidth": 1,
        "borderColor": "#ccc",
        "marginBottom": 15,
        "paddingBottom": 15,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    allowFontScaling={true}
    maxLength={15}
    onChangeText={[Function]}
    placeholder="Password"
    rejectResponderTermination={true}
    secureTextEntry={true}
    style={
      Object {
        "alignSelf": "center",
        "borderBottomWidth": 1,
        "borderColor": "#ccc",
        "marginBottom": 15,
        "paddingBottom": 15,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
            Object {
              "color": "#3740FE",
            },
          ]
        }
      >
        Sign up
      </Text>
    </View>
  </View>
  <Text
    onPress={[Function]}
    style={
      Object {
        "color": "#3740FE",
        "marginTop": 25,
        "textAlign": "center",
      }
    }
  >
    Already registered? Click here to login!
  </Text>
</View>
`;
